### ISSUE TITLE
Default keyboard: Focus can't exit workspace using Tab or Shift+Tab in JavaScript and Python editors

### ISSUE THREAD

**Issue by Nina-1210** [microsoft/pxt-microbit#6859]
**Describe the bug**
In the JavaScript and Python editors, keyboard navigation is constrained within the code workspace. When focus enters the workspace, users cannot navigate out to adjacent UI sections using the Tab or Shift+Tab keys, resulting in a keyboard trap.

1. Navigate to https://makecode.microbit.org/beta.
2. Open an existing project or create a new project.
3. Switch to either the **JavaScript** or **Python** editor.
4. Press **Tab** until focus enters the code workspace.
5. Continue pressing **Tab** or **Shift+Tab** to attempt to move focus out of the workspace.

**Actual behavior**
Focus remains trapped within the workspace. Users cannot navigate to other UI elements using Tab or Shift+Tab.
 
**Expected behavior**
Focus should move out of the workspace and proceed to the next or previous focusable UI element when using Tab or Shift+Tab, enabling full keyboard navigation without trapping.

**Screen Recording**
https://github.com/user-attachments/assets/83c643cb-bc02-4eb0-9977-7d42e470c639

**Additional context**
1.OS: Windows 11 25H2
2.Browser: Chrome, Edge, and Firefox
3.makecode.microbit.org version: 8.1.16
4.Microsoft MakeCode version: 12.3.11

**Comment by microbit-robert** [comment]
This is typical text editor behaviour and there is usually some kind of escape hatch to allow users to tab backwards and forwards. This is documented in https://makecode.com/accessibility:
> JavaScript and Python editor navigation
> These keyboard shortcuts are used in the JavaScript editor:

> By default, pressing Tab in the editor will insert the tab character. Toggle this behavior by pressing Control+M on Windows or Control+Shift+M on Mac. In order to jump to the toolbox from the editor. Press Control+Alt+T on Windows or ⌘+Alt+T on Mac.

**Comment by srietkerk** [comment]
It looks like the Control+M mechanic works. Is the suggestion here to have the tab navigation be the default with the text editors? This would mean that anyone wanting to tab while coding would first have to hit Control+M to do so, which would be a pretty big change.

**Comment by srietkerk** [comment]
Chatted with the team about it, changing this behavior is undesirable. Since the toggle is already documented, we are keeping the current behavior.
